examples.test_fail.MyTestClass.test_find_army_of_robots_on_xkcd_desert_island
----------------------------------------------------
Last Page: https://xkcd.com/731/
  Browser: firefox
Timestamp: 1599068455  (Unix Timestamp)
     Date: Wednesday, September 2, 2020
     Time: 1:40:55 PM  (EDT, UTC-05:00)
----------------------------------------------------
Traceback: File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 615, in run
    testMethod()
  File "/Users/michael/github/SeleniumBase/examples/test_fail.py", line 17, in test_find_army_of_robots_on_xkcd_desert_island
    self.assert_element("div#ARMY_OF_ROBOTS", timeout=1)
  File "/Users/michael/github/SeleniumBase/seleniumbase/fixtures/base_case.py", line 4931, in assert_element
    self.wait_for_element_visible(selector, by=by, timeout=timeout)
  File "/Users/michael/github/SeleniumBase/seleniumbase/fixtures/base_case.py", line 3290, in wait_for_element_visible
    self.driver, selector, by, timeout)
  File "/Users/michael/github/SeleniumBase/seleniumbase/fixtures/page_actions.py", line 299, in wait_for_element_visible
    timeout_exception(NoSuchElementException, message)
  File "/Users/michael/github/SeleniumBase/seleniumbase/fixtures/page_actions.py", line 117, in timeout_exception
    raise exc(message)
Exception: Message: 
 Element {div#ARMY_OF_ROBOTS} was not present after 1 second!
